ci/cloudbuild.yaml

```diff
@@ -1,8 +1,8 @@
 steps:
 
-- name: gcr.io/cloud-builders/git
+- name: 'gcr.io/cloud-builders/git'
   id: 'Fetch tags from git'
-  args: [fetch, --depth=100, --tags]
+  args: [fetch, --unshallow]
```

Comment:
IIRC this will fetch all the commits from the beginning of time. Correct? Is this something we want?